src/components/TimeCardPage/CommentModal.tsx

```diff
@@ -0,0 +1,85 @@
+//import { Menu, MenuButton, MenuItem } from '@aws-amplify/ui-react';
+import {
+    Input, Button, useDisclosure, Modal,
+    ModalContent, ModalHeader, VStack, Card, CardBody, StackDivider, ModalFooter
+} from '@chakra-ui/react'
+import { useFormik } from "formik";
+import React from 'react';
+
+export default function CommentModal(props) {
+
+    // enum for the type of comment the manager is leaving
+    enum CommentType {
+        Report,
+        Comment,
+    }
```

Comment:
Ny convention, let's also pull this out into a `types.ts` file for easy reference + reuse